start-chrome-for-playr.sh

```diff
@@ -58,7 +144,21 @@ else
 	reload_url=file://$2
 fi
 
+# Only preference file adjustments are needed on Linux
+if [ "$(uname)" == "Linux" ]; then
+	# Prevent popups or additional empty tabs
+	sed -i -E 's/("exited_cleanly":\s*)false/\1true/g' $preferences_file || true
+	sed -i -E 's/("exit_type":\s*)"Crashed"/\1"Normal"/g' $preferences_file || true
+fi
+
 # to check the values of the variables created above uncomment the following line
 # echo "file://"${playr_loader_file}"?channel="${channel}"&reload_url="${reload_url}
 # the --app= option prevents the "Restore pages" popup from showing up after the previous process was killed
-$browser ${gpu_options} ${persistency_options} ${no_nagging_options} --kiosk --app="file://"${playr_loader_file}"?channel="${channel}"&reload_url="${reload_url}
+if [ "$(uname)" == "Darwin" ]; then
```

Comment:
als we ook andere browsers willen ondersteunen moeten we dit iets complexer maken aangezien de command line flags dan anders zullen zijn